collector/collector.go

```diff
@@ -52,11 +52,11 @@ func parseStatus(data sql.RawBytes) (float64, bool) {
 	if bytes.Equal(data, []byte("Connecting")) {
 		return 0, true
 	}
-	// SHOW GLOBAL STATUS like 'wsrep_cluster_status' can return "Primary" or "Non-Primary"/"Disconnected"
+	// SHOW GLOBAL STATUS like 'wsrep_cluster_status' can return "Primary" or "non-Primary"/"Disconnected"
 	if bytes.Equal(data, []byte("Primary")) {
 		return 1, true
 	}
-	if bytes.Equal(data, []byte("Non-Primary")) || bytes.Equal(data, []byte("Disconnected")) {
+	if bytes.Equal(data, []byte("non-Primary")) || bytes.Equal(data, []byte("Disconnected")) {
```

Comment:
Looking at the [documentation](https://galeracluster.com/library/documentation/monitoring-cluster.html)

> The node should only return a value of Primary. Any other value indicates that the node is part of a nonoperational component. This occurs in cases of multiple membership changes that result in a loss of quorum or in cases of split-brain situations.

I think we should change this block to be an `else`. Anything that doesn't match "Primary" should return 0.

Reply:
CC @rtreffer, what do you think?

Reply:
Oh, nevermind, we have to do it this way otherwise it short-circuits the function. :frowning_face: